### ISSUE TITLE
lua 51, lua 52, lua53 and luajit

### ISSUE THREAD

**Issue by onodera-punpun** [6c37/crux-ports#62]
In responese to https://github.com/6c37/crux-ports/commit/acfebf9958a83aac65e8e96726659073946afa77 and https://github.com/6c37/crux-ports/commit/22e9617d7c71a3b7d2c476625b24100866de28cc

now some packages depends on two or even three different lua versions, wound't it be better for the user to make that choice? This means adding a README, and putting all the different lua implementations in optional:.

**Comment by rtlanceroad** [comment]
Yeah. I agree. i see arch forks built all the lua versions for lua-related pkg if pkg is compatible. But this idea is great.

**Comment by onodera-punpun** [comment]
On what lua implementation should we make it depend by default? I'm in favor of luajit.

**Comment by rtlanceroad** [comment]
According to [performance](http://luajit.org/performance.html) , we should use luajit to be faster :-)

**Comment by kori** [comment]
Luajit is a drop-in replacement for lua 5.2, by the way.
